components/job-orchestration/job_orchestration/scheduler/compress/compression_scheduler.py

```diff
@@ -78,6 +80,78 @@ def update_compression_job_metadata(db_cursor, job_id, kv):
     db_cursor.execute(query, values)
 
 
+def _process_fs_input_paths(
+    fs_input_conf: FsInputConfig, paths_to_compress_buffer: PathsToCompressBuffer
+) -> None:
+    """
+    Iterate through all files in fs_input_conf and adds their metadata to the
+    paths_to_compress_buffer.
+    Note: this method skips any files that do not exist.
+    :param fs_input_conf: FS configuration specifying the files to compress.
+    :param paths_to_compress_buffer: PathsToCompressBuffer containing the scheduling information
```

Comment:
```suggestion
    :param fs_input_conf:
    :param paths_to_compress_buffer:
```

I think the variable names + types is sufficient. What do you think?